app/src/main/java/com/kickstarter/viewmodels/ProjectCardHolderViewModel.java

```diff
@@ -0,0 +1,279 @@
+import com.kickstarter.libs.rx.transformers.Transformers;
+import com.kickstarter.libs.utils.BooleanUtils;
+import com.kickstarter.libs.utils.NumberUtils;
+import com.kickstarter.libs.utils.ObjectUtils;
+import com.kickstarter.libs.utils.ProgressBarUtils;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Category;
+import com.kickstarter.models.Photo;
+import com.kickstarter.models.Project;
+import com.kickstarter.models.User;
+import com.kickstarter.ui.viewholders.ProjectCardViewHolder;
+
+import org.joda.time.DateTime;
+
+import java.util.List;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.coalesce;
+
+public interface ProjectCardHolderViewModel {
+
+  interface Inputs {
+    /** Call to configure the view model with a project. */
+    void configureWith(Project project);
+    void projectClicked();
+  }
+
+  interface Outputs {
+    /** Emits the string name of the projects category. */
+    Observable<String> backersCountText();
+    Observable<Boolean> backingViewGroupIsGone();
+    Observable<String> categoryNameText();
+    Observable<String> deadlineCountdownText();
+    Observable<Boolean> featuredViewGroupIsGone();
+    Observable<List<User>> friendsForNamepile();
+    Observable<String> friendAvatarUrl();
+    Observable<Boolean> imageIsInvisible();
+    Observable<Boolean> friendBackingViewIsHidden();
+    Observable<Boolean> fundingUnsuccessfulTextViewIsGone();
+    Observable<Boolean> metadataViewGroupIsGone();
+    Observable<Project> projectOutput();
+    Observable<Integer> percentageFunded();
+    Observable<Boolean> percentageFundedProgressBarIsGone();
+    Observable<String> percentageFundedText();
+    Observable<String> photoUrl();
+    Observable<String> blurbText();
+    Observable<String> nameText();
+    Observable<Project> notifyDelegateOfProjectClick();
+    Observable<Boolean> potdViewGroupIsGone();
+    Observable<DateTime> projectCanceledAt();
+    Observable<DateTime> projectFailedAt();
+    Observable<Boolean> projectStateViewGroupIsGone();
+    Observable<DateTime> projectSuccessfulAt();
+    Observable<DateTime> projectSuspendedAt();
+    Observable<String> rootCategoryName();
+    Observable<Void> setDefaultTopPadding();
+    Observable<Void> setMetadataTopPadding();
+    Observable<Boolean> starredViewGroupIsGone();
+    Observable<Boolean> successfullyFundedTextViewIsGone();
+  }
+
+  final class ViewModel extends ActivityViewModel<ProjectCardViewHolder> implements Inputs, Outputs {
+
+    public ViewModel(final @NonNull Environment environment) {
+      super(environment);
+
+      final Observable<Boolean> projectInEndedStates = this.project
+        .map(
+          p -> p.state().equals(Project.STATE_SUCCESSFUL)
+            || p.state().equals(Project.STATE_CANCELED)
+            || p.state().equals(Project.STATE_FAILED)
+            || p.state().equals(Project.STATE_SUSPENDED)
```

Comment:
while looking through that deadline stuff I came across this `ProjectUtils::isCompleted` helper which does precisely this! https://github.com/kickstarter/android-oss/blob/888a3746835835016fc1a1bb32d8e2a90b8bffce/app/src/main/java/com/kickstarter/libs/utils/ProjectUtils.java#L83-L89

if we statically import we could even do `this.project.map(isCompleted)` :D